closed/GensrcJ9JCL.gmk

```diff
@@ -51,8 +51,11 @@ $(eval $(call SetupCopyFiles,COPY_OVERLAY_FILES, \
 	SRC := $(TOPDIR), \
 	DEST := $(SUPPORT_OUTPUTDIR)/overlay, \
 	FILES := \
+		src/java.base/share/classes/java/lang/ClassValue.java \
+		src/java.base/share/classes/java/lang/invoke/ClassSpecializer.java \
 		src/java.base/share/classes/java/security/Security.java \
 		src/java.base/share/classes/java/util/Timer.java \
+		src/java.base/share/classes/java/util/concurrent/ConcurrentHashMap.java \
```

Comment:
Please use the same order as in https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/128.

Reply:
Redo cherry-pick from https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/128